client/telecomUniverseComponents/validator/index.js

```diff
@@ -0,0 +1,199 @@
+import angular from 'angular';
+import _ from 'lodash';
+import punycode from 'punycode';
```

Comment:
`punycode` must also be removed from the globals on the `.eslintrc` file.

ref: https://github.com/ovh-ux/ovh-manager-telecom/blob/master/.eslintrc#L19

pay attention that `$ yarn run lint:js` is still green.

Reply:
Done.